HospitalFrontend/src/views/CreateSurveyResponse.vue

```diff
@@ -0,0 +1,161 @@
+                    <div class="form-check form-check-inline">
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id" 
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 1)">
+                        <label class="form-check-label">1</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 2)">
+                        <label class="form-check-label">2</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 3)">
+                        <label class="form-check-label">3</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 4)">
+                        <label class="form-check-label">4</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 5)">
+                        <label class="form-check-label">5</label>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <button type="button" class="btn btn-success btn-lg mb-5" v-on:click="saveSurveyResponse">Submit survey</button>
+    </div>
+
+    
+</template>
+
+<script>
+
+import axios from 'axios'
+import api from '../constant/api.js'
+import Toastify from 'toastify-js'
+
+
+
+export default {
+
+    name:'CreateSurveyResponse',
+    data:function() {
+        return{          
+            survey: null,
+            surveyResponse: {
+                surveyId:null,
+                ratedSurveySections:[],
+                doctorSurveySection:null,
+                patientAccountId:1
+            }
+        }
+    },
+    methods: {
+        getSurvey: function () {
+            axios.get(api.survey + '/find/1').then((response)=>{
+                this.survey = response.data;
+                this.surveyResponse.surveyId = this.survey.id
+            })
+        },
+        rateQuestion: function (section, questionId, rating) {
+            if (section.isDoctorSection) {
+                if (this.surveyResponse.doctorSurveySection == null) {
+                    this.surveyResponse.doctorSurveySection = {
+                        doctorId: 1,
+                        surveySectionId: section.id,
+                        ratedSurveyQuestions: [{
+                            surveyQuestionId: questionId,
+                            rating: rating
+                        }]
```

Comment:
I'm not sure I understand the logic behind this function.

Reply:
Unfortunately, simplification of this function would require some model changes which we intend to do in future sprints.For now, I can tell you it has been tested properly.

Reply:
What 'doctorId: 1' represents?

Reply:
It's a hardcoded doctorId which represents a doctor about whom the survey is about. We have no forms for creating new surveys or doctors, that's why it is hardcoded right now.